frontend/src/components/Sidebar/Sidebar.tsx

```diff
@@ -186,6 +186,15 @@ export default function Sidebar() {
 
   const search = namespaces.size !== 0 ? `?namespace=${[...namespaces].join('+')}` : '';
 
+  const handleToggleOpen = useCallback(() => {
+    dispatch(setWhetherSidebarOpen(!sidebar.isSidebarOpen));
+  }, [sidebar.isSidebarOpen]);
+
+  const linkArea = useMemo(
+    () => <DefaultLinkArea sidebarName={sidebar.selected.sidebar || ''} isOpen={isOpen} />,
+    [sidebar.selected.sidebar, isOpen]
+  );
```

Comment:
If we are using a memo here, do we need the memo around the DefaultLinkArea?

Reply:
linkArea is passed as a prop, without this it triggers rerender for PureSidebar 

![image](https://github.com/user-attachments/assets/80aa4399-4325-48c0-b38b-e686542f51e7)

And the DefaultLinkArea component is memoized to avoid rerenders inside PureSidebar for unrelated rerenders
(on the screenshot DefaultLinkArea is _c3 and memoized)
![image](https://github.com/user-attachments/assets/b8fd535a-da03-4302-9857-e0352dcadc77)